packages/polling/src/ratelimiter.ts

```diff
@@ -84,6 +87,11 @@ export abstract class RateLimiter<T, U> implements IRateLimiter<T, U> {
     return this.poll.stop();
   }
 
+  /**
+   * Arguments for the underlying function.
+   */
+  protected args: V | undefined = undefined;
```

Comment:
Can we default `args` to `[]` so that the factory can call `fn(...this.args)` directly?

Reply:
This works even with it set to `undefined`.